Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -154,13 +169,19 @@ public async Task ValidateContestPassword(int id, bool official, string password
             return participationModel;
         }
 
+        private void SetValidationError(ValidationResult? validationResult, ContestValidationModel validationModel)
+        {
+            var propertyInfo = validationModel.GetType().GetProperty(validationResult.PropertyName);
```

Comment:
Cant you use automapper for this?

Reply:
The ContestValidationModel has properties of type bool and the ValidationResult has properties of type string. I don't think I can use the automapper to map the properties of the validation model from the validation result.